### PR TITLE
Update napari plugin entries

### PR DESCRIPTION
**Pull request by zoccoler**
- updated entries 3 and 4 (mostly 3) with Thalley Lambert's suggestions from napari zulipchat;
- qt gui automatically updates when changes in designer are made;
- magicgui version now can provide active slider (slider as output);
- using auto_call =  True now;
- updated related screenshots and code in scripts.

### CODE REVIEW

**Comment by zoccoler** [comment]
Hi @jo-mueller, this is not urgent, but whenever you can, can you please take a look if all is OK before I merge? :)

File: marcelo_zoccoler/entry_user_interf3/Readme.md [hunk 1]
```diff
@@ -255,26 +263,33 @@ viewer.window.add_dock_widget(flood_widget, area='right')         # Add our gui
...
 flood_widget = magicgui(flood, delta={'label': 'Temperature Increase (Δ°C):', 
                                            'min': 0, 'max' : 3, 'step': 0.1},
                                 new_level={'label':'Sea Level (dm):', 'widget_type':'Slider',
-                                           'min': 0, 'max' : 255})
+                                           'min': 0, 'max' : 255, 'enabled' : False},
+                                auto_call=True)
 ```
 
-Now, you get practically the same result as the earlier approach. If you don't know the widget options, take a look [here](https://napari.org/magicgui/usage/widget_overview.html) to find the widget you want and discover its parameters.
+Now, you get practically the same result as the earlier approach. We even discarded the button by using `auto_call=True`. If you don't know the widget options, take a look [here](https://napari.org/magicgui/usage/widget_overview.html) to find the widget you want and discover more about its parameters.
```
Comment (jo-mueller) [inline marcelo_zoccoler/entry_user_interf3/Readme.md:276]: So if I get it correctly, `auto_call=True` will automatically call the associated functions whenever a widgets is changed/clicked/etc? Maybe you could point this out a bit more clearly - if the associated function is not very fast, using `auto_call` could lead to trouble and probably Napari freezing until all queued calls to the function behind the plugin are processed?

**Review by jo-mueller — APPROVED** [review]
Looks good to me. Just one minor comment with respect to using `auto_call=True`.

File: marcelo_zoccoler/entry_user_interf3/Readme.md (hunk 1 above)
Comment (zoccoler) [reply marcelo_zoccoler/entry_user_interf3/Readme.md]: Good catch! I updated the text to warn the reader about that possibility.

**Review by jo-mueller — COMMENTED** [review]
Looks good to me 👍

**Pull request merged**